src/server/scripts/Northrend/zone_borean_tundra.cpp

```diff
@@ -229,15 +229,16 @@ class npc_khunok_the_behemoth : public CreatureScript
 ## npc_corastrasza
 ######*/
 
-#define GOSSIP_ITEM_C_1 "I... I think so..."
-
 enum Corastrasza
 {
-    SPELL_SUMMON_WYRMREST_SKYTALON               = 61240,
-    SPELL_WYRMREST_SKYTALON_RIDE_PERIODIC        = 61244,
+    GOSSIP_ITEM_I_I_TINK_SO                = 0,
```

Comment:
*think

Reply:
Thanks. I really was tired when I put this together.